### DISCUSSION TITLE
Bitcraze VM

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1126] [Q&A]
Hi, Does there have anyone doing the VM in the macOS Arm architecture ? I donllowed the Virtual Box and 2023.10 this VM version in the mac but appear some errors. tks

**Comment by knmcguire** [comment]
hi! We have seen issues with apple computers and virtual box (see the last comment on [this discussion](https://github.com/orgs/bitcraze/discussions/688)). It is not sure if it has been solved yet though and we need time to investigate this.

In the meantime we advise you to install the software tools directly on your mac computer instead of using the VM

**Reply by knmcguire** [reply]
I've made an issues for the VM repo so we won't forget about this again: https://github.com/bitcraze/bitcraze-vm/issues/68

**Reply by CalvinKoung** [reply]
Hi, so in the meantimes I can't  use the VM client in the MAC (M1) laptop right ? Because I want to use client to set the loco position system anchors and through the other tools to know the crazyflie data and state. These are currently not possible right ? tks.

**Reply by knmcguire** [reply]
There are instructions to setup the cfclient for apple m1's so that should be possible: https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/installation/install/#mac. I think probably the same instructions apply for the [LPS tools](https://github.com/bitcraze/lps-tools) as well I hope?

**Reply by CalvinKoung** [reply]
Hi, I followed the instruction but when I doing the second step, it will stuck. 
(arch --x86_64 brew install python@3.9 libusb) this command.

**Reply by knmcguire** [reply]
@CalvinKoung could you start a new discussion for that? That would be more suitable as this no longer is about the VM

**Reply by CalvinKoung** [reply]
ok

**Reply by knmcguire** [reply]
It is just so that it will be easier for us to help you out with that particular problem, especially since it should be able to work with no problem to work natively with m1 apples and if not we should definitely figure out what is wrong

**Comment by ataffanel** [comment]
Virtualbox for apple silicon is still beta/dev preview. I just checked and I confirm that the VM is not booting. It is unlikely we can do anything on our side so we will just make sure we document that the VM does not work on apple-silicon macs. The way forward then is to follow Kimberly's advice: install tools natively.

Hopefully we can find a better solution in the future. I am currently looking at vsCode dev-container which would allow to develop in a docker container and this would likely work on all platform including apple-silicon macs.
